merlinx.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "acex88",
+			"oem": "Redmi",
+			"device": "Redmi Note 9",
+			"filename": "crDroidAndroid-14.0-20231229-merlinx-v10.0.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/merlinx/10.x/crDroidAndroid-14.0-20231229-merlinx-v10.0.zip/download",
+			"timestamp": 1703815341,
+			"md5": "8c1dc1512348bf59af6fe608dec956c9",
+			"sha256": "df972d59b44b80df91f2e2d2c72baed801ec0c3181a1f21180eac39c2c917562",
+			"size": 1133653976,
+			"version": "10.0",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-14-official-crdroid-v10-aosp-stable.4634565/#post-89234949",
```

Comment:
include follow back url for donation page on forum
also remove #post-89234949 as it will link to specific post on that forum